Application/SchedulingLib/models/DatabaseModels.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SchedulingLib.models
+{
+    public class Building
+    {
+        public int BuildingId { get; set; }
+        public string BuildingName { get; set; }
+
+        public List<Rooms> Rooms { get; set; }
```

Comment:
Make sure the types are correct in this file. `List<Rooms>` should be `List<Room>`, `List<Sections>` should be `List<Section>`, etc.